lib/membrane_h264_ffmpeg/parser.ex

```diff
@@ -266,38 +264,41 @@ defmodule Membrane.H264.FFmpeg.Parser do
       state,
       next_bufs,
       resolution_changes,
-      caps ++ buffers_before_change ++ pending_caps ++ acc
+      stream_format ++ buffers_before_change ++ pending_stream_format ++ acc
     )
   end
 
   @impl true
-  def handle_caps(:input, %Membrane.H264.RemoteStream{}, ctx, _state)
+  def handle_stream_format(:input, %Membrane.H264.RemoteStream{}, ctx, _state)
       when ctx.pads.input.start_of_stream?,
-      do: raise("Cannot send Membrane.H264.RemoteStream caps after the stream has started")
+      do:
+        raise("Cannot send Membrane.H264.RemoteStream stream_format after the stream has started")
```

Comment:
In addition to reformatting, I'd rephrase the error a bit:
```suggestion
    raise "Cannot handle Membrane.H264.RemoteStream format after the stream has started"
  end
```